migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql

```diff
@@ -0,0 +1,116 @@
+
+-- 1. spuSOLRTreatmentEntryLevel (validated in snowflake)
+CREATE OR REPLACE PROCEDURE DEV.SP_LOAD_SOLRTREATMENTENTRYLEVEL() 
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS CALLER
+    AS  
+
+DECLARE
+
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+    
+-- SOLRTreatmentEntryLevel depends on : Show.SOLRTreatmentEntryLevel, Base.SpecialtyToCondition, Base.TreatmentLevel, Base.SpecialtyToProcedureMedical, Base.Specialty, and Base.MedicalTerm
```

Comment:
One row per table for readability
```suggestion
-- SOLRTreatmentEntryLevel depends on :
---- Show.SOLRTreatmentEntryLevel
---- Base.SpecialtyToCondition
---- Base.TreatmentLevel
---- Base.SpecialtyToProcedureMedical
---- Base.Specialty
---- Base.MedicalTerm
```

Reply:
Done